assets/javascripts/initializers/discourse-reactions.js.es6

```diff
@@ -49,6 +50,41 @@ function initializeDiscourseReactions(api) {
       post
     });
   });
+
+  api.modifyClass("component:emoji-value-list", {
+    didReceiveAttrs() {
+      this._super(...arguments);
+
+      if (this.setting.setting !== "discourse_reactions_enabled_reactions") {
+        return;
+      }
+
+      let defaultValue = this.values.includes(
+        this.siteSettings.discourse_reactions_like_icon
+      );
+
+      if (!defaultValue) {
+        this.collection.unshift({
+          emojiUrl: emojiUrlFor(
+            this.siteSettings.discourse_reactions_like_icon
+          ),
+          isEditable: false,
+          isEditing: false,
+          showUpDownButtons: this.collection.length - 1 ? true : false,
```

Comment:
when is this used? when do we stop showing up/down buttons?

Reply:
We made a computedproperty for this, ill remove this from here